etc/yoda/component-config.json

```diff
@@ -2,6 +2,7 @@
   "paths": ["/usr/yoda/lib/component"],
   "interception": {
     "runtimeDidLogin": [ "ota.runtimeDidLogin" ],
+    "runtimeDidWakeup": [ "customConfig.runtimeDidWakeup" ],
```

Comment:
Wondering if it's appropriate to name the `wakeup` from hibernation such similar to `turenDidWakeup` 🤔

Reply:
How about `didResumeFromSleep`

Reply:
Updated the event name.